campaignion_email_to_target/messages_app/drupal-fixture.js

```diff
@@ -134,21 +135,19 @@ var Drupal = {
    *   A translated string.
    */
   formatPlural: function (count, singular, plural, args, options) {
-    var args = args || {};
-    args['@count'] = count;
+    var args = args || {}
+    args['@count'] = count
     // Determine the index of the plural form.
-    var index = Drupal.locale.pluralFormula ? Drupal.locale.pluralFormula(args['@count']) : ((args['@count'] == 1) ? 0 : 1);
+    var index = Drupal.locale.pluralFormula ? Drupal.locale.pluralFormula(args['@count']) : ((args['@count'] == 1) ? 0 : 1)
 
     if (index == 0) {
-      return Drupal.t(singular, args, options);
-    }
-    else if (index == 1) {
-      return Drupal.t(plural, args, options);
-    }
-    else {
-      args['@count[' + index + ']'] = args['@count'];
-      delete args['@count'];
-      return Drupal.t(plural.replace('@count', '@count[' + index + ']'), args, options);
+      return Drupal.t(singular, args, options)
+    } else if (index == 1) {
```

Comment:
Please use [brace-style](https://eslint.org/docs/latest/rules/brace-style) stroustrup for Drupal JS.

Reply:
Just added a commit to change this, and then I thought a bit... this is not Drupal JS, it's just a partial copy of Drupal JS that is never used in a Drupal context. It's just used by the Vue app in dev and testing mode where the `Drupal` global isn't available.
Also, the Vue app's linter stripped away all the semicolons, which might not fit the Drupal style.
So in my opinion it's cleaner to have this file linted like the rest of the Vue app. This would make my last commit obsolete.